src/containers/collaborations/components/DataSetListCard.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import React from 'react';
+
+import styled from 'styled-components';
+import { faChevronRight } from '@fortawesome/pro-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Map, fromJS, get } from 'immutable';
+import { Button, Colors } from 'lattice-ui-kit';
+import { ValidationUtils, useGoToRoute } from 'lattice-utils';
+import type { UUID } from 'lattice';
+
+import {
+  DataSetTitle,
+  GapGrid,
+  SpaceBetweenGrid,
+} from '../../../components';
+import { Routes } from '../../../core/router';
+import { ID, ORGANIZATION_ID } from '../../../utils/constants';
+
+const { isValidUUID } = ValidationUtils;
+
+const { NEUTRAL } = Colors;
+
+const BasicListCard = styled.div`
+  align-items: center;
+  background-color: ${NEUTRAL.N50};
+  border-radius: 5px;
+  cursor: pointer;
+  display: flex;
+  padding: 8px 16px;
```

Comment:
I feel like we have a BasicListCard that exists somewhere already. Maybe we can consolidate?

Reply:
I think we have this styled component. I can pull it out.